phpstan-baseline.neon

```
@@ -180,6 +192,24 @@ parameters:
 			count: 1
 			path: src/Bundle/Doctrine/DBAL/Driver.php
 
+		-
+			message: '#^Call to an undefined method Doctrine\\DBAL\\Query\\Expression\\ExpressionBuilder\:\:andX\(\)\.$#'
+			identifier: method.notFound
+			count: 1
+			path: src/Bundle/Doctrine/DBAL/ExpressionBuilder.php
+
+		-
+			message: '#^Call to an undefined method Doctrine\\DBAL\\Query\\Expression\\ExpressionBuilder\:\:orX\(\)\.$#'
+			identifier: method.notFound
+			count: 1
+			path: src/Bundle/Doctrine/DBAL/ExpressionBuilder.php
+
+		-
+			message: '#^Parameter \#3 \$y of method Doctrine\\DBAL\\Query\\Expression\\ExpressionBuilder\:\:comparison\(\) expects string, mixed given\.$#'
+			identifier: argument.type
+			count: 1
+			path: src/Bundle/Doctrine/DBAL/ExpressionBuilder.php
+
```

Same also here, each of those new ignored errors are fixable by adding PHPDoc